schedule/yast/maintenance/qam-yast_self_update_sle15sp3_dev.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: qam-yast_self_update
+description: installation using self_update as boot parameter
+vars:
+  YUI_REST_API: 1
+schedule:
+  license_agreement:
+    - installation/validate_self_update
```

Comment:
here is the same situation, I would suggest to create new hook in default file due to it is not related with license.